src/Runtime/HttpKernel.php

```diff
@@ -43,15 +43,16 @@ public function handle(Request $request)
     {
         $this->app->useStoragePath(StorageDirectories::PATH);
 
+        $kernel = $this->resolveKernel($request);
+
         if (static::shouldSendMaintenanceModeResponse($request)) {
             $response = new Response(
-                file_get_contents($_ENV['LAMBDA_TASK_ROOT'].'/503.html'), 503
+                file_get_contents($_ENV['LAMBDA_TASK_ROOT'].'/503.html'),
+                503
             );
 
-            $this->app->terminate();
+            $this->app->instance('middleware.disable', true);
```

Comment:
Can you share some info on why this was added here?

Reply:
I was assuming that the request avoid terminable middleware but now that I think about it, if the request isn't going through `Kernel::handle()` at all the terminating middleware callbacks will never be applied here:

https://github.com/laravel/framework/blob/7.x/src/Illuminate/Foundation/Http/Kernel.php#L178

Shall I remove?

Reply:
Hmm, can you explain more? Why should it avoid terminable middleware?

Reply:
From my perspective, if I were using terminable middleware within an app I'd assume that the request made it all the way through the app. But since the response is being hijacked by the 503.html file and the request is not being handled by the kernel (not going through the router) it seemed odd to still call terminable middleware.